include/openenclave/bits/device.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_BITS_DEVICE_H
+#define _OE_BITS_DEVICE_H
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+#include <openenclave/bits/types.h>
+
+OE_EXTERNC_BEGIN
+
+/**
+ * Device identifiers known by Open Enclave.
+ */
+enum
+{
+    /** The null device id. */
+    OE_DEVID_NONE,
+
+    /** The standard input device. */
+    OE_DEVID_STDIN,
+
+    /** The standard output device. */
+    OE_DEVID_STDOUT,
+
+    /** The standard errord evice. */
+    OE_DEVID_STDERR,
+
+    /** The non-secure host file system. */
+    OE_DEVID_HOSTFS,
+
+    /** The Intel SGX protected file system. */
+    OE_DEVID_SGXFS,
+
+    /** The non-secure host socket device. */
+    OE_DEVID_HOSTSOCK,
```

Comment:
Unless this token matches an ANSI or POSIX standard, don't use abbreviations like "sock", spell out as "socket"

Reply:
Thanks Dave! Fixed all of these.